src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("Search Results", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(<SearchResults results={[]} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders with results", () => {
+    const results = ["search-image1", "search-image2", "search-image3"];
+    const { asFragment } = render(<SearchResults results={results} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders no results message when there are no results", () => {
+    render(<SearchResults results={[]} hasSearched={true} />);
+    const noResultsMessage = screen.getByText(
+      "No results" && "Please try another search"
+    );
+    expect(noResultsMessage).toBeInTheDocument();
```

Comment:
This test is falsely passing. To check whether this is happening in any test, change the text you are searching for to something that definitely is not present and run it again. If it still passes, you know something is wrong!
The reason is you cant use operators to search for two strings like you are here in your query. As these 2 bits of content are in separate elements, you need to search for them both separately and them run assertions on each of them like this:
```
    render(<SearchResults results={[]} hasSearched={true} />);
    const noResultsMessage = screen.getByText("No results");
    const anotherSearchMessage = screen.getByText("Please try another search");

    expect(noResultsMessage).toBeInTheDocument();
    expect(anotherSearchMessage).toBeInTheDocument();
```